### PR TITLE
Support default value of generic parameter

### PR DESCRIPTION
**Pull request by taichi-ishitani**
resolve veryl-lang/veryl#718

### CODE REVIEW

File: crates/analyzer/src/namespace.rs [hunk 1]
```diff
@@ -47,6 +47,20 @@ impl Namespace {
         }
     }
 
+    pub fn is_subset(&self, x: &Namespace) -> bool {
```
[the remark this answers is not in the document]
Comment (taichi-ishitani) [reply crates/analyzer/src/namespace.rs]: Use `included` method instead of `is_subset`.

File: crates/analyzer/src/analyzer_error.rs [hunk 2]
```diff
@@ -227,6 +227,21 @@ pub enum AnalyzerError {
...
     },
 
+    #[diagnostic(
+        severity(Error),
+        code(missing_default_argument),
+        help("give default argument"),
+        url("")
+    )]
+    #[error("missing default argument for parameter {identifier}")]
+    MissingDeefaultArgument {
```
Comment (dalance) [inline crates/analyzer/src/analyzer_error.rs:237]: Maybe typo?

```suggestion
    MissingDefaultArgument {
```

Comment (taichi-ishitani) [reply crates/analyzer/src/analyzer_error.rs]: I've fixed it.

**Comment by dalance** [comment]
Looks good!
I'll merge this PR.

**Pull request merged**